src/core/stream/types.ts

```diff
@@ -88,5 +88,14 @@ export interface IStreamLoadedEvent {
   type : "loaded";
   value : true;
 }
+export interface IStreamPlayingEvent {
+  type : "playing";
+  value : true;
+}
+
+export interface IStreamPausedEvent {
+  type: "paused";
+  value: true;
```

Comment:
Why true? (I know loaded is `true` too, but that make no sense either)

More `undefined` or `null` no?